README.md

```diff
@@ -70,3 +70,4 @@ What if you don't know what kind of website to "just build"? The following is a
 * Limit yourself to a palette of black-and-a-color, or white-and-a-color
 * Choose another restraint and create web art inspired by it
 * Make a time calculator ([Requirements](docs/time-calculator-requirements.md))
+* Make a unit converter ([Requirements](docs/unit-converter-requirements.md))
```

Comment:
The Google converter was just a suggestion if the user wasn't sure how to design the converter. Not only the look/layout of it, but the functionality of it as well. For example, when the main topic changes (length, area, speed, etc), the _from_ and _to_ labels and drop down options change along with it. That's some good JS practice.

I think recreating the Google converter would be quite advanced. The math behind some of those conversions could get a little hairy.